addons/base_iban/models/res_partner_bank.py

```diff
@@ -28,6 +28,30 @@ def get_bban_from_iban(iban):
     """
     return normalize_iban(iban)[4:]
 
+def get_iban_part(iban, number_kind):
+    """ Get part of the iban depending on the mask
+        .. code-block:: python
+            # template = 'ITkk KBBB BBSS SSSC CCCC CCCC CCC'
+            partner.acc_number = 'IT60X0542811101000000123456'
+            get_iban_part(partner.acc_number, 'bank') == '05428'
+            get_iban_part(partner.acc_number,'account') == '000000123456'
+        Returns ``False`` in case of failure
+    """
+    kind_map = {
```

Comment:
`kind_map` seems like a weird name, I'm expecting a nice map here :smile: `iban_part` instead of `kind`?
The other possibility is to not do this mapping and ask the user to put directly `get_iban_part(iban, 'B')`.
I'm not sure we're really gaining much by doing this mapping (for `bank` it's obvious, but for `check`, anyway people won't know the mapping we use, I think)